dvc/tree/azure.py

```diff
@@ -92,6 +93,10 @@ def blob_service(self):
             container_client.get_container_properties()
         except ResourceNotFoundError:
             container_client.create_container()
+        except HttpResponseError as e:
+            # client may not have account-level privileges
+            if e.status_code != 403:
```

Comment:
```suggestion
            if exc.status_code != 403:
```

Reply:
Just for consistency and to not annoy the linter :)

Reply:
No problem :)